### DISCUSSION TITLE
Amasty Request a Quote Pro

### DISCUSSION THREAD

**Discussion by sanjeev-ca** [breezefront/community#186] [3. Integrations]
Purchase Amasty Request a Quote. It does not work with breezefront theme. Please make this theme compatible to this module.

**Comment by dimdimich** [comment]
Sorry for delay with response. Can you please first try to follow https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/ and contact Amasty and let us know theor response. We had disucssion with Amasty and they said that in case they will have enough requests they will consider adding integration from their side. If they will not do that please let us know and we will add it to our todo list.
